pkgs/development/python-modules/coconut/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, buildPythonPackage
+, fetchurl
+, prompt-toolkit
+, pygments
+, pyparsing
+}:
+
+buildPythonPackage rec {
```

Comment:
Isn't this an application rather than a library?

Reply:
I think it is both (two different ways to call it - as a python import or the command line).

Reply:
Alright. Then it can stay that way.